spec/lib/submission_builder/ty2022/states/return_header_spec.rb

```diff
@@ -154,9 +154,9 @@
       )
     }
     let(:primary_signature_pin) { "12345" }
-    let(:primary_esigned_at) { Time.now.strftime("%Y-%m-%d") }
+    let(:primary_esigned_at) { DateTime.tomorrow.beginning_of_day } # Setting esigned_at to midnight UTC time to check if the date is correctly filled into the XML as the day before in EST time
```

Comment:
[dust] To make this test setup more clear about what it is testing for, I suggest making it a separate test with a case like "when the filer esigns after midnight UTC but not after midnight in the State's timezone"